spec/controllers/comments_controller_spec.rb

```diff
@@ -24,4 +24,46 @@
       expect(response).to have_http_status :not_found
     end
   end
+
+  describe "DELETE comment" do
+    context "with authenticated users" do
+
+      before do
+        request.env["HTTP_REFERER"] = 'http://localhost/'
+      end
```

Comment:
I have a redirect_to back in the controller, so it needs the ref or the test won't work.